components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppLauncherActivity.kt

```diff
@@ -67,7 +54,10 @@ class WebAppLauncherActivity : AppCompatActivity() {
         when (manifest?.display) {
             WebAppManifest.DisplayMode.FULLSCREEN,
             WebAppManifest.DisplayMode.STANDALONE,
-            WebAppManifest.DisplayMode.MINIMAL_UI -> launchWebAppShell(startUrl)
+            WebAppManifest.DisplayMode.MINIMAL_UI -> {
+                emitHomescreenIconTapFact()
```

Comment:
You can leave `emitHomescreenIconTapFact` in `launchWebAppShell` where it was instead. I'm fine with either location.